docker/cloudbuild.yaml

```diff
@@ -36,7 +36,12 @@ steps:
     # The following trick is from https://stackoverflow.com/a/52400857/90297 with great gratuity
     echo '{"version": "3.4", "networks":{"default":{"external":{"name":"cloudbuild"}}}}' > docker-compose.override.yml
     ./install.sh
-    ./test.sh || docker-compose logs nginx web relay
+    if ! ./test.sh; then
+      echo "Test failed.";
+      docker-compose ps;
+      docker-compose logs nginx web relay;
+      exit -1;
```

Comment:
Do docs suggest exiting -1? This isn't even a valid exit code, and actually wraps around to mean 255.

Reply:
Oh, right. I'll make that 5 or something.